snuba/datasets/sessions_processor.py

```diff
@@ -27,6 +27,18 @@
 metrics = MetricsWrapper(environment.metrics, "sessions.processor")
 
 
+def enforce_retention_days(message: Mapping[str, Any]) -> int:
+    retention_days: int = message["retention_days"]
+
+    if settings.ENFORCE_RETENTION:
+        retention_days = (
+            settings.LOWER_RETENTION_DAYS
+            if retention_days <= settings.LOWER_RETENTION_DAYS
+            else settings.DEFAULT_RETENTION_DAYS
+        )
+    return retention_days
```

Comment:
Since this has become common code for most datasets, might be worth putting in a place where common code lives. Maybe the utils directory? You can just pass in the retention day number to the util function so that different datasets could have different keys and it would still work.

Reply:
yeah I was actually going to just use the `enforce_retention` that is used in the events_formats